README.md

```diff
@@ -616,87 +616,89 @@ tool to generate a contributors graphic below.
+      <td align="center"><a href="https://github.com/tacaswell"><img src="https://avatars.githubusercontent.com/u/199813?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Thomas A Caswell</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=tacaswell" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/AlfredoR-CSUF"><img src="https://avatars.githubusercontent.com/u/86503312?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Alfredo Rodriguez</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=AlfredoR-CSUF" title="Code">💻</a></td>
+    </tr>
+    <tr>
+      <td align="center"><a href="https://github.com/dylanmcreynolds"><img src="https://avatars.githubusercontent.com/u/40469975?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dylan McReynolds</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=dylanmcreynolds" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/jbteves"><img src="https://avatars.githubusercontent.com/u/26722533?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Joshua Teves</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=jbteves" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/chrisvam"><img src="https://avatars.githubusercontent.com/u/6949021?v=4?s=100" width="100px;" alt=""/><br /><sub><b>chrisvam</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=chrisvam" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/maffettone"><img src="https://avatars.githubusercontent.com/u/43007690?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dr. Phil Maffettone</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=maffettone" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/lmilechin"><img src="https://avatars.githubusercontent.com/u/13963460?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Lauren Milechin</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=lmilechin" title="Code">💻</a></td>
+      <td align="center"><a href="adamcooper.sh"><img src="https://avatars.githubusercontent.com/u/14908880?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Adam Cooper</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=super-cooper" title="Code">💻</a></td>
+      <td align="center"><a href="http://www.rafmudaf.com"><img src="https://avatars.githubusercontent.com/u/13797903?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Rafael M Mudafort</b></sub></a><br /><a href="https://github.com/USRSE/usrse.github.io/commits?author=rafmudaf" title="Code">💻</a></td>
+    </tr>
+  </tobdy>
```

Comment:
```suggestion
  </tbody>
```
I don't know what changed where but this seems a regression.

Reply:
I guess we'll have to this over and over again until we fix the source that's generating this...